### PR TITLE
Add basic, unergonomic CLI

### PR DESCRIPTION
**Pull request by tomjw64**
Runnable right now with `cd bakery` then `yarn node ./src/cli/execute.js --help`

`execute.js` and the task file changes are what is of interest. I am aware there is a lot of yarn dependency noise in here.

### CODE REVIEW

**Comment by tomjw64** [comment]
@ripal what version of yarn are you using?

**Comment by rnathuji** [comment]
I'm currently using yarn version 1.22.0 and node version 13.9.0.

**Comment by tomjw64** [comment]
Interesting. I'm using yarn berry (v2). I believe that's part of the problem. I might convert the repo back to npm because this is turning out to be more trouble than it's worth.

**Comment by tomjw64** [comment]
@ripal let me know if this works for you (you may have to clean out some things and `npm install`)

**Comment by rnathuji** [comment]
@tomjw64 - Yes, things are working now, thanks!

File: bakery/src/cli/execute.js [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+  })
+  .option('d', {
+    alias: 'data',
+    demandOption: true,
+    describe: 'path to data directory',
+    normalize: true,
+    type: 'string'
+  })
+  .demandCommand(1, 'command required')
+  // .usage(`Usage: ${process.env.CALLER || 'execute.js'} <command>`)
```
Comment (rnathuji) [inline bakery/src/cli/execute.js:515]: Is this commented line perhaps a stray from dev and removable?

**Pull request merged**